src/rdgnu-backend/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
is there a reason this needs its own eslintrc?

Reply:
This and the .gitignore were created by default when I did the create-nest-app. I will remove and add to the project level stuff :)